kalite/settings/base.py

```diff
@@ -322,6 +323,9 @@
 
 ROOT_URLCONF = "kalite.distributed.urls"
 
+BACKUP_DIRPATH = os.path.join(expanduser("~"), 'ka-lite-backups')
```

Comment:
I don't think it should be placed there by default, rather in the configured home directory, stored in `_data_path`.